### ISSUE TITLE
Mac OS X support

### ISSUE THREAD

**Issue by exe0cdc** [PySCeS/pysces#27]
Currently the only mention of installation on OS X is in the INSTALL.txt file (and the source archives on the downloads page), but it seems that these instructions might be outdated (e.g. Enthought Python Distribution is now called Enthought Canopy). I am relatively unfamiliar with OS X, so I am asking the following questions from the perspective of a user that wants to install PySCeS on a "new/clean" OS X system with the least amount of fuss:

- Is Enthought Canopy still the recommended distribution, or is Anaconda preferred? 
- What compilers are required/recommended for Mac OS X, and how does one go about getting them?
- Does ``pip install pysces`` work if the compilers are present?
- OS X comes with Python 2.7. Can PySCeS be installed (with pip or otherwise) without a Python distribution?
- Is it possible to get a binary package for PySCeS on OS X?

**Comment by bgoli** [comment]
Hi, I don't have a Mac so can't help with binaries but as far as I know the following should work:

- Use Anaconda Python if possible (it includes conda package management)
- On OS X install XCode and the command line utilities, afaik this includes everything you need to compile stuff
- You can use the native Python but then you need to install/build all the dependencies yourself, I would try Anaconda 
- If you get PySCeS built and working with Anaconda on OS X then we can look at build binaries :-)

**Comment by exe0cdc** [comment]
Thanks Brett,

I will test building and installation on a Mac by next week Friday and report back.

**Comment by exe0cdc** [comment]
This is a quite late, but I have tested installation of PySCeS on a Mac using Anaconda. I wrote instructions for installing PySCeS, which are included in PySCeSToolbox's [documentation](http://pyscestoolbox.readthedocs.io/en/latest/installation.html#id2).

Basically the xcode command line tools are needed (it's not necessary to install the full xcode), as well as the gfortran binaries from http://gcc.gnu.org/wiki/GFortranBinaries . From there `pip install pysces` works as expected.
